<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>snippetManager</class>
 <widget class="QWidget" name="snippetManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="parm_inp_search"/>
     </item>
     <item>
      <widget class="QTableWidget" name="parm_tbl_snippets"/>
     </item>
     <item>
      <widget class="QPushButton" name="parm_btn_load">
       <property name="text">
        <string>Load</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
